doc/110-Installation/30-API-Check-Forwarder.md

```diff
@@ -106,3 +106,70 @@ Enable-IcingaFrameworkApiChecks;
 ## EventLog Errors
 
 In case a check could not be executed by using this experimental feature, either because of timeouts or other issues, they are added with `EventId 1553` inside the EventLog for `Icinga for Windows`. A description on why the check could not be executed is added within the event output.
+
+## Icinga Communication to API
+
+With Icinga 2.14.0 and later, you can enable the Icinga Agent to natively communicate with the Icinga for Windows API, allowing checks being executed without having to start a PowerShell.
```

Comment:
Also, I guess this will be the link to our docs:

https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#ifw-api

Reply:
https://icinga.com/docs/icinga-2/snapshot/doc/10-icinga-template-library/#ifw-api
already works thanks to @bobapple